src/app/pages/connectors/connectors-table/connectors-table.component.ts

```diff
@@ -0,0 +1,90 @@
+import { Component, OnInit } from '@angular/core';
+import { LocalDataSource } from 'ng2-smart-table';
+import { ConnectorConfigService } from 'src/app/@core/services/connector-config.service';
+import {
+  JSONRenderComponent,
+  TickCrossExtraRenderComponent,
+  TickCrossRenderComponent,
+} from 'src/app/@theme/components/smart-table/smart-table';
+
+@Component({
+  templateUrl: './connectors-table.component.html',
+})
+export class ConnectorsTableComponent implements OnInit {
+  // ng2-smart-table data source
+  tableSource: LocalDataSource = new LocalDataSource();
+
+  // ng2-smart-table settings
+  settings = {
+    actions: {
+      add: false,
+      edit: false,
+      delete: false,
+    },
+    pager: {
+      display: true,
+      perPage: 10,
+    },
+    columns: {
+      name: {
+        title: 'Name',
+        filter: false,
+      },
+      description: {
+        title: 'Description',
+        filter: false,
+      },
+      disabled: {
+        title: 'Active',
+        filter: false,
+        type: 'custom',
+        valuePrepareFunction: (c, r) => {
+          return !c; // disabled = !active
+        },
+        renderComponent: TickCrossRenderComponent,
+      },
+      config: {
+        title: 'Configurations Added',
+        filter: false,
+        type: 'custom',
+        renderComponent: JSONRenderComponent,
+      },
+      missing_secrets: {
+        title: 'Missing Secrets',
+        filter: false,
+        type: 'custom',
+        valuePrepareFunction: (c, r) => {
+          return r.verification.missing_secrets;
+        },
+        renderComponent: JSONRenderComponent,
+      },
+      configured: {
+        title: 'Configured',
+        filter: false,
+        type: 'custom',
+        valuePrepareFunction: (c, r) => {
+          return {
```

Comment:
No need of extra `return` and curly brackets, would be cleaner if done as an inline arrow function.